postgres/problems/problem_07.md

```
@@ -1,21 +1,23 @@
+* GROUP BY を使用すること
+* WHERE, HAVING, JOIN, ORDER BY などの追加句は使用しないこと
 
 ## 想定出力例（先頭 5 行）
-
-| id | name            | price |
-|----|-----------------|-------|
-| 23 | High-end Laptop |240000 |
-| 45 | 4K Monitor      | 85000 |
-| …  | …              |  …   |
+ 
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)